test/cli/template-use/common-dir/in/repository/_common/foo/bar/extractor/ex-generic-v2/myconfig/lib/part1.jsonnet

```diff
@@ -0,0 +1,3 @@
+{
+    key1: "value1",
```

Comment:
Zostal tu posledny problem, a to, ze v tomto importovanom subore nie su definovane funkcie, napr. `Input()`, to vyriesim v dalsom PR.